src/server/server.js

```diff
@@ -46,8 +48,11 @@ app.use(expressWinston.logger({
 app.use(helmet());
 app.use(session(sessionConfig(conf)));
 app.use(Express.static(__dirname + '/../public', { maxAge: '365d' }));
+const metricsBundle = promBundle({ autoregister: false });
+app.use(metricsBundle);
 
 // routes
+app.use('/metrics', trustedNetworks, metricsBundle.metricsMiddleware);
```

Comment:
Does this mean we only allow metrics to be called from internal network, not from anyone?